src/main/frontend/src/components/landingpage/LowerSection.js

```diff
@@ -0,0 +1,19 @@
+import InfiniteScrollComponent from "./InfiniteScrollComponent";
+import classes from "./LowerSection.module.css";
+
+function MiddleSection() {
+  return (
+    <div className={classes.section_container}>
+      <div className={classes.section_title_div}>
+        <button className={classes.option_button_selected}>New Arrivals</button>
+        <button className={classes.option_button}>Top Rated</button>
+        <button className={classes.option_button}>Last Chance</button>
```

Comment:
How do you plan integrating different content inside InfiniteScrollComponent based on which button is clicked?